docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -32,7 +32,11 @@ For more information about KPA and HPA, see the documentation on [Supported Auto
         metadata:
           annotations:
             autoscaling.knative.dev/metric: "concurrency"
+            autoscaling.knative.dev/target-utilization-percentage: "70" 
     ```
+!!! note <br/>
```

Comment:
```suggestion
            autoscaling.knative.dev/target-utilization-percentage: "70" 
    ```
    !!! note <br/>
```

Reply:
Thanks @psschwei , I have compared the existing page and changes looks fine to me. Kindly review. Thanks again for your help!